.github/workflows/test.yml

```diff
@@ -35,41 +35,19 @@ jobs:
           git clone --depth 1 git://github.com/pyvista/gl-ci-helpers.git
           powershell gl-ci-helpers/appveyor/install_opengl.ps1
 
-      - name: Get pip cache dir
-        id: pip-cache
-        run: |
-          echo "::set-output name=dir::$(pip cache dir)"
-
-      - name: pip cache
-        uses: actions/cache@v2
-        with:
-          path: ${{ steps.pip-cache.outputs.dir }}
-          key: ${{ runner.os }}-pip-${{ matrix.python-version }}-${{ hashFiles('**/setup.cfg') }}
-          restore-keys: |
-            ${{ runner.os }}-pip-${{ matrix.python-version }}-
-            ${{ runner.os }}-pip-
-
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
           pip install -e .[testing]
-          pip install pytest-cov
-
-      - uses: actions/cache@v2
-        id: cache
-        with:
-          path: micromanager_gui/Micro-Manager-*
-          key: ${{ hashFiles('micromanager_gui/install_mm.py') }}
```

Comment:
Any reason not to do a monthly//weekly expiring cache like in pymmcore-plus? https://github.com/tlambert03/pymmcore-plus/blob/19cf062150aea77953f2864d7738fb7ccc3c2ec3/.github/workflows/test_and_deploy.yml#L39-L50